src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml.cs

```diff
@@ -626,5 +629,32 @@ private void UpdateZoomScaleValueLabel(Slider slider, Label label)
                 label.Content = slider.Value.ToString() + "%";
             }
         }
+
+        private void GraphicHelpersScalingSlider_DragCompleted(object sender, DragCompletedEventArgs e)
+        {
+            Slider slider = (Slider)sender;
+            UpdateGraphicScaleValueLabel(slider, lblHelpersScalingValue);
+        }
+
+        private void UpdateGraphicScaleValueLabel(Slider slider, Label label, bool initialRun = false)
+        {
+            double value = slider.Value;
+            if (label != null)
+            {
+                //The margin value for the label goes from - 480 to 310, resulting in 790 pixels from the starting point to the end.
+                //The value is decreased to 480 because the margin begins at - 480
+                //This is the relation between the margin in pixels and the value
+                double marginValue = (790 * value / 10000) - 480;
```

Comment:
I think any kind of pixel logic is going to need robust testing in various DPI settings - theres probably a way to factor in wpf scale value here.